base_local_planner/package.xml

```diff
@@ -22,7 +22,6 @@
     <build_depend>cmake_modules</build_depend>
     <build_depend>message_generation</build_depend>
     <build_depend>tf2_geometry_msgs</build_depend>
-    <build_depend>quadruped_msgs</build_depend>
```

Comment:
No update to CMakeLists.txt?

Reply:
This package is already in `<depend>` tag, so `<build_depend>` is not necessary. Therefore removed it here, but it remains in `CMakeLists.txt`.